src/sockaddr.rs

```diff
@@ -49,6 +49,22 @@ impl SockAddrStorage {
     /// # Safety
     ///
     /// The type `T` must be one of the `sockaddr_*` types defined by this platform.
+    ///
+    /// # Examples
+    /// ```
+    /// # #[allow(dead_code)]
```

Comment:
I'm pretty sure dead-code warnings are disabled by default in documentation tests.